src/code/audio_synthesis.c

```diff
@@ -459,12 +458,12 @@ void func_800DBE64(void) {
 void func_800DBE6C(void) {
 }
 
-void AudioSynth_LoadFilter(Acmd* cmd, s32 flags, s32 countOrBuf, void* addr) {
-    aFilter(cmd, flags, countOrBuf, addr);
+void AudioSynth_LoadFilterBuffer(Acmd* cmd, s32 flags, s32 buf, void* addr) {
+    aFilter(cmd, flags, buf, addr);
```

Comment:
Why not the generic `countOrBuf` ?

Reply:
This was a matter of naming this function as how it's used. The flags for filtering should be:
`A_CONTINUE` -> 0x0
`A_INIT` -> 0x1
`A_SETUP`/`A_SIZE` -> 0x2
So the way this function is used in practice, it seems to be for flags `A_CONTINUE`/`A_INIT`, which is paired with `AudioSynth_LoadFilterSize` for `A_SETUP` (aFilter needs to be called multiple times to be properly used. it's its own setup function).

So when using these functions, it seems the intent
```
AudioSynth_LoadFilterSize()
AudioSynth_LoadFilterBuffer()
```
So it seems like `AudioSynth_LoadFilterBuffer` wasn't really intented to hold a count. Of course because of the flags argument, you could put a size in there if you want, but it seems that what `AudioSynth_LoadFilterSize` is suppose to do.

Does that make sense? It's hard to explain

Reply:
yeah I think I get it now thanks